src/features/camera.md

```diff
@@ -0,0 +1,240 @@
+# Cameras
+
+{{#include ../include/links.md}}
+
+Cameras are what drives all rendering in Bevy. They are responsible for
+configuring what to draw, how to draw it, and where to draw it.
+
+You must have at least one camera entity, in order for anything to be displayed
+at all! If you forget to spawn a camera, you will get an empty black screen.
+
+In the simplest case, you can create a camera with the default settings. Just
+spawn an entity using [`Camera2dBundle`][bevy::Camera2dBundle] or
+[`Camera3dBundle`][bevy::Camera3dBundle]. It will simply draw all renderable
+entities that are [visible][cb::visibility].
+
+This page gives a general overview of cameras in Bevy. Also see the dedicated
+pages for [2D cameras][cb::camera-2d] and [3D cameras][cb::camera-3d].
+
+Practical advice: always create [marker components][cb::component-marker] for
```

Comment:
This is no longer needed IME: `Camera2d` / `Camera3d` exist and UI cmaeras are no more.

Reply:
I disagree. If you have more than one camera, you are probably using them for different purposes (game camera, overlay camera, whatever), so you should have marker components.

If you are just prototyping / starting out and have only one camera... well... i think it's good practice to make a marker component anyway, so you don't accidentally end up with broken logic if you ever want to use more cameras later.